src/components/home/LatestTransactions.blocks.tsx

```diff
@@ -0,0 +1,160 @@
+import { useTranslation } from 'react-i18next';
+import { ConfirmedTransactionData } from '@ardenthq/sdk/distribution/esm/confirmed-transaction.dto.contract';
+import cn from 'classnames';
+import { ReactNode } from 'react';
+import dayjs from 'dayjs';
+import { getTimeAgo } from '../../lib/utils/getTimeAgo';
+import { EmptyConnectionsIcon, Icon, IconDefinition, Tooltip } from '@/shared/components';
+import { usePrimaryWallet } from '@/lib/hooks/usePrimaryWallet';
+import Amount from '@/components/wallet/Amount';
+import trimAddress from '@/lib/utils/trimAddress';
+
+export const NoTransactions = () => {
+  const { t } = useTranslation();
+
+  return (
+    <div className="mt-12 flex flex-col items-center justify-center gap-6">
+      <EmptyConnectionsIcon />
+      <div className='max-w-40 leading-tight text-base font-normal text-center dark:text-white'>
+        {t('PAGES.HOME.NO_TRANSACTIONS')}
+      </div>
+    </div>
+  );
+};
+
+enum TransactionType {
+    SEND = 'send',
+    RECEIVE = 'receive',
+    RETURN = 'return',
+    SWAP = 'swap',
+    VOTE = 'vote',
+    UNVOTE = 'unvote',
+    SECOND_SIGNATURE = 'second-signature',
+    MULTISIGNATURE = 'multisignature',
+    REGISTRATION = 'registration',
+    RESIGNATION = 'resignation',
+    OTHER = 'other',
+}
+
+const TransactionListItem = ({ transaction }: { transaction: ConfirmedTransactionData }) => {
+    const primaryWallet = usePrimaryWallet();
+    const { t } = useTranslation();
+
+    const getType = (transaction: ConfirmedTransactionData): string => {
+        if (transaction.isTransfer()) {
+            const isSender = transaction.sender() === primaryWallet?.address();
+            const isRecipient = transaction.recipient() === primaryWallet?.address();
+        
+            if (isSender && isRecipient) {
```

Comment:
if you have a look at how the [`isSent()`](https://github.com/ArdentHQ/platform-sdk/blob/develop/packages/ark/source/confirmed-transaction.dto.ts#L73) and [`isReceived()`](https://github.com/ArdentHQ/platform-sdk/blob/develop/packages/ark/source/confirmed-transaction.dto.ts#L77) methods work in the sdk, you will notice that they rely on metadata that needs to be set for them to determine this state. Since that metadata doesn't exist, it will return `false` 

You can mimic this locally by manually setting the metadata before calling the methods
```
transaction.setMeta('address', primaryWallet?.address());
transaction.setMeta('publicKey', primaryWallet?.publicKey());
```

You'll then notice that the methods provide the proper values. However, the underlying issue is with the way you retrieve the transactions. If you take a closer look at the PSDK, you will see that `setMeta` is used here https://github.com/ArdentHQ/platform-sdk/blob/develop/packages/profiles/source/transaction-index.ts#L67 when fetching transactions. I'll continue this on the line where you fetch the transactions currently